syntaxes/lit-html-string-injection.json

```diff
@@ -0,0 +1,10 @@
+{
+	"fileTypes": [],
+	"injectionSelector": "L:source.js (meta.tag), L:source.jsx (meta.tag), L:source.js.jsx (meta.tag), L:source.ts (meta.tag), L:source.tsx (meta.tag)",
+	"patterns": [
```

Comment:
This approach looks good. My only concerns is that `meta.tag` is alway used by jsx, so this selector may effect react code too

I think a safer selector would be:

```
"injectionSelector": "L:meta.embedded.block.html meta.tag",
```

Can you try that to see if works

Reply:
Good idea, and thanks for the suggestion!

I tested on `highlighting.test.js` and a local Typescript project, which both seems to work well. Unfortunately I don't have any handy JSX projects to look at, but the `meta.embedded.block.html` should only be applied for `html`` ` tagged template string literals if I understand the `lit-html.json` file correctly.